src/ui/components.tsx

```diff
@@ -116,6 +130,18 @@ export const CheckMkGenericAsyncSelect = function <Value extends string | (strin
     onChange(newValue.value);
   };
 
+  const formatOptionLabel = React.useCallback((option: SelectableValue<Value>, meta: unknown) => {
```

Comment:
The second argument seems unnecessary to me. Shouldn't typescript be happy about it being left out?

Reply:
seems to work without